client/components/SongArt.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SongArt = ({
+  songImage, handleAlbumClick,
+}) => {
+  const handleClick = (e) => {
+    e.preventDefault();
+    const type = (e.target.id === 'player-song-artist') ? 'Artist' : 'Album';
+    handleAlbumClick();
```

Comment:
If the artist was clicked, would we still want there to be a console log of 'album clicked'?

Reply:
This is something that I am still trying to figure out how to handle... I am not sure whether I want to have a SongArt method that pops up and alert, or have the method live in Player.jsx. I think it will end up being the latter.